app/views/companies/_search_form.html.haml

```diff
@@ -0,0 +1,49 @@
+= search_form_for @search_params, method: :get, url: companies_path,
+                                        class: 'form-horizontal' do |f|
+  .form-group
+    .col-sm-5
+      = f.label :business_categories_id_in,
+                "#{t('activerecord.models.business_category.one')}",
+                class: 'control-label',
+                style: 'text-align: left; font-size: 12px;'
+      = f.collection_select :business_categories_id_in,
```

Comment:
@weedySeaDragon - absolutely ... I've been meaning to add a story about adding dev/test data creation to the seeds.rb file (unless we think there is a better place/way to do that?)